meta/main.yml

```diff
@@ -30,6 +30,6 @@ galaxy_info:
         - 2012
         - 2012R2
         - 2016
-  categories:
+  galaxy_tags:
```

Comment:
Is this compatible with the oldest ansible version we support?

Reply:
2.5, yes: https://docs.ansible.com/ansible/2.5/reference_appendices/galaxy.html?highlight=galaxy_tags#import-a-role as far as I can see

Reply:
👍